dsl-reference.md

```diff
@@ -1416,6 +1417,33 @@ from:
 to: '.petList += [ . ]'
 ```
 
+### Export
+
+Certain task needs to set the context of the workflow to save the task output for later usage. Users set the content of the context through a runtime expression. The result of the expression is the new value of the context. The expression is evaluated against the existing context. 
+
+Optionally, the context might have an associated schema. 
+
+#### Properties
+
+| Property | Type | Required | Description |
+|----------|:----:|:--------:|-------------|
+| schema | [`schema`](#schema) | `no` | The [`schema`](#schema) used to describe and validate context.<br>*Included to handle the non frequent case in which the context has a known format.* |
+| as | `string`<br>`object` | `no` | A runtime expression, if any, used to export the output data to the context. |
```

Comment:
Link is missing on `runtime expression` -> `[runtime expression](#runtime-expressions)`